<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>group_dashboardClass</class>
 <widget class="QWidget" name="group_dashboardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分组</string>
  </property>
  <widget class="QPushButton" name="m_btnGroupConfirm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>95</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnGroupCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>95</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_edtGroupName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>210</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
